apps/mull-ui/src/app/pages/home/home-discover.scss

```diff
@@ -0,0 +1,14 @@
+.react-swipeable-view-container > div[aria-hidden='true'] {
+  max-height: calc(100vh - 11.5rem);
+  overflow-y: hidden !important; // hides scroll bar when changing routes. !important needed to overwrite the element style set by react swipeable views package.
```

Comment:
I don't see the y-scrollbar when changing from the discover route to upcoming route or my events route unless it happens in a certain browser, I'm using google chrome. Also, there's probably a way to avoid using !important and override the react swipeable view package by using a class name or some property

Reply:
Did you have enough events to scroll through? It did make a difference for me.
As for the !important, I needed it because there's inline style on the component and this can only be overridden by classes when you use `!important`

This is one of the acceptable cases to use `!important`, here's some documentation on it, checkout out the "How !important can be used:" section